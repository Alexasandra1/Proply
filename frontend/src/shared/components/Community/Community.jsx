import React from 'react';
import './Community.scss';

export function Community(props) {
    return (
        <div className={`communityBlock ${props.styleCommunity}`}>
            <img src={props.image} alt="community" className={`community__image ${props.styleImage}`} />
            <div className="community__textBlock">
                <p className={`name ${props.styleName}`} >{props.name}</p>
                <p className={`texComm ${props.styleText}`}>{props.text}</p>
            </div>
        </div>
    );
}
