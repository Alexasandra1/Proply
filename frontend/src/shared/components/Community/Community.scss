.communityBlock {
    display: flex;
    text-align: center;
    flex-direction: column;
    align-items: baseline;
}

.community__image {
    width: 19.25rem;
    height: 26.875rem;
}

.name {
    color: #111;
    font-family: Montserrat;
    font-size: 2.25rem;
    font-style: normal;
    font-weight: 500;
}

.styleTextComm{
    color: #A7A7A7;
    font-family: Montserrat;
    font-size: 1.5rem;
    font-style: normal;
    font-weight: 500;
}