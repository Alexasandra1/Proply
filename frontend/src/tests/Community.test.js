import React from 'react';
import renderer from 'react-test-renderer';
import { Community } from '../shared/components/Community/Community';

test('renders correctly', () => {
  const props = {
    styleCommunity: 'test-style',
    image: 'test-image.jpg',
    styleImage: 'test-image-style',
    styleName: 'test-name-style',
    name: 'Test Name',
    styleText: 'test-text-style',
    text: 'Test Text',
  };

  const tree = renderer
    .create(<Community {...props} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
