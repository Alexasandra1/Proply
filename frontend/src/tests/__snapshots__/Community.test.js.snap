// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="communityBlock test-style"
>
  <img
    alt="community"
    className="community__image test-image-style"
    src="test-image.jpg"
  />
  <div
    className="community__textBlock"
  >
    <p
      className="name test-name-style"
    >
      Test Name
    </p>
    <p
      className="texComm test-text-style"
    >
      Test Text
    </p>
  </div>
</div>
`;
